cmd/bnsapi/docs/docs.go

```diff
@@ -363,6 +363,33 @@ var doc = `{
                 }
             }
         },
+        "/msgfee/msgfees": {
+            "get": {
+                "description": "Return message fee information based on message path: account/first_mesage",
```

Comment:
This endpoint should be returning global message fees, eg `username/register_token`.  Fees at the domain level should be provided by the `/account/domains/` endpoint.